### DISCUSSION TITLE
Issue with sign text while using plugin

### DISCUSSION THREAD

**Discussion by L1ghtningStrife** [GriefPrevention/GriefPrevention#2161] [Q&A]
While placing signs on a modded server I'm setting up, all the text changes to literal{the message} for each row. Figured out that it was this plugin and unsure as to how to fix it and would really appreciate any help :)
![image_2023-10-04_203720039](https://github.com/TechFortress/GriefPrevention/assets/114982281/5ba454ed-9ecc-4728-b833-77cc6489242b)

**Comment by QarthO** [comment]
Hmmm, this is a new one for me, haven't seen something like this before

How did you find out it was GriefPrevention?

Also modded API doesn't perfectly mesh well with the Bukkit API, so some issues could come from your server software.
I'd make sure you're on the correct GP version for the specific modded version you're on. Apart from that, you might just be out of luck. Updates/Bugfixes won't really come for older versions

**Reply by QarthO** [reply]
Looking at the code for anything sign related, you might be able to bypass sign handling by disabling sign notifications in the config.

If you're still having this issue after changing the config, then it's either not GriefPrevention, or there's an issue with your server software not handling the the bukkit API correctly that GP calls

Update: What version of GP are you on? Maybe sign handling was different back then

**Reply by L1ghtningStrife** [reply]
Thank you for your quick response! I found out which plugin it was using the good ol' fashion uninstall and reinstall one by one until the problem fixes lol. I wasn't sure about the version so I just reinstalled 16.18.1 and the problem's still there, I already had sign notifications disabled in configs. It might genuinely be an API issue like you mentioned, I don't suppose there's a version with just the land claims module? We're using EssentialsX for moderation and were just planning to use this for the land claim system because I loved how it was set up.
